recipes/nodejs_from_chocolatey.rb

```diff
@@ -0,0 +1,24 @@
+#
+# Author:: Hossein Margani (hossein@margani.dev)
+# Cookbook:: nodejs
+# Recipe:: install_from_chocolatey
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+include_recipe 'chocolatey'
+
+chocolatey_package 'nodejs-lts' do
+  version node['nodejs']['version'] if node['nodejs']['version']
+  action :upgrade
```

Comment:
Is the upgrade action required?

Reply:
Without the upgrade action chocolatey will not install if an older version is already installed. In chocolatey “upgrade” means “install or upgrade if the target version (or newer) is not installed”